doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -576,28 +525,30 @@ Linear Algebra Equivalents
      - inverse Fourier transform of ``a``
 
    * - ``sort(a)``
-     - ``sort(a)`` or ``a.sort()``
-     - sort the matrix
+     - ``sort(a)`` or ``a.sort(axis=0)``
+     - sort each column of a 2D matrix, ``a``
 
-   * - ``[b,I] = sortrows(a,i)``
```

Comment:
I think this one is still worth keeping

Reply:
I can put this back in, but the description needs an update. This is not sorting rows, its sorting the entire array by row based upon a column number, `i`.  e.g.

```matlab
A=[1,2,3;3,2,1;2,3,1]
sortrows(A,1)
ans =

   1   2   3
   2   3   1
   3   2   1
```
vs

```python
>>> A = np.array([[1,2,3],[3,2,1],[2,3,1]])
>>> I = argsort(A[:,0])
>>> A[I,:]
    array([[1, 2, 3],
           [2, 3, 1],
           [3, 2, 1]])
```

Reply:
I added the `sortrows` for matlab again, but distinguished it from a typical sort:

* ``[b,I]=sortrows(a,1)``
* ``I=argsort(a[:,0]); b=a[I,:]``

`save the array ``a`` as array ``b`` with rows sorted by the first column`